pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,6 +69,7 @@ func NewCmdLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		}
 		cmdutil.CheckErr(o.RunLog())
 	}
+	cmd.Flags().String("version", "", "View the logs of the deployment associated with the provided version for a deployment config")
```

Comment:
This is currently an `int`, I'd like to see it as an `int` until we have a reason to loosen it.